lib/LedgerSMB/Scripts/import_csv.pm

```diff
@@ -352,14 +356,23 @@ sub _process_sic {
 sub _process_timecard {
     use LedgerSMB::Timecard;
     my ($request, $entries) = @_;
-    my $myconfig = {};
-    my $jc = {};
+    my @floats = qw {qty non_billable sellprice allocated};
     for my $entry (@$entries) {
+        my $jc = {};
         my $counter = 0;
         for my $col (@{$cols->{timecard}}){
+            if ($request->{sep} eq ";" &&
+                any { $_ eq $col } @floats) {
```

Comment:
Floats are using comma instead of period when the CSV separator is the semi-column